### ISSUE TITLE
full_crypto feature for sp-core

### ISSUE THREAD

**Issue by fzzr-** [paritytech/subxt#325]
Please enable `full_crypto` feature for sp-core.

**Comment by jsdw** [comment]
If you'd like that feature, you can depend on `sp_core` explicitly and enable that feature for yourself; I think that this is the best way to give you control over which features you want without bloating `subxt` with either extra dependencies that may not be needed, or it's own feature flags to enable dependency features.

**Comment by ascjones** [comment]
Yes the `sp_core` is only re-exported for use by the macro, and we do not require that feature for the generated code. As @jsdw suggests, you should be able to add a dependency to it in your own crate.
